industrial_ci/ci_main.sh

```diff
@@ -82,16 +93,18 @@ BUILDER=catkin
 ROSWS=wstool
 # For compatibilility with hydro catkin, which has no --verbose flag
 CATKIN_TEST_RESULTS_CMD="catkin_test_results"
-if [ catkin_test_results --verbose 1>/dev/null 2>/dev/null; then CATKIN_TEST_RESULTS_CMD="catkin_test_results --verbose"; fi
```

Comment:
If you really want to turn off verbosity, introduce a `VERBOSE_TEST_RESULTS` variable which defaults to `true`